human_aware_rl/imitation/behavior_cloning_tf2_test.py

```diff
@@ -112,7 +111,8 @@ def test_lstm_evaluation(self):
         results = evaluate_bc_model(model, self.bc_params)
 
         # Sanity Check
-        self.assertGreaterEqual(results, 20.0)
+        # Currently not run since training lstm takes too much compute for github CI
+        # self.assertGreaterEqual(results, 20.0)
```

Comment:
I personally don't think we should comment out assert statements like this. If the test is indeed crashing the github servers, then we could add a flag to the argparse that indicates whether to execute this test or not so github knows not to run it. Even though we're really not using the LSTM code right now, I worry that commenting out the asserts like this could give the false impression that things are working and the test has passed, when it hasn't

Reply:
Adderssed